webapp/index.html

```diff
@@ -17,72 +17,39 @@
         <script src="lib/angulartics-ga.js"></script>
         <script src="lib/angular-facebook.js"></script>
         <script src="script-config.js"></script>
-        <script src="script.js"></script>
+        <script src="app/app.js"></script>
```

Comment:
seems like adding all the script tags here (+ making sure they are in the right order) will get annoying as time goes on. we should look into using something like http://requirejs.org/ in the future.